website/docs/d/network_service_tags.html.markdown

```diff
@@ -0,0 +1,51 @@
+---
+subcategory: "Network"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_network_service_tags"
+description: |-
+  Gets information about Service Tags for a specific service type.
+---
+
+# Data Source: azurerm__network_service_tags
+
+Use this data source to access information about Service Tags.
+
+## Example Usage
+
+```hcl
+data "azurerm_network_service_tags" "example" {
+  location        = "West Europe"
+  service         = "AzureKeyVault"
+  location_filter = "northeurope"
+}
+
+output "address_prefixes" {
+  value = data.azurerm_service_tags.example.address_prefixes
+}
+```
+
+## Arguments Reference
+
+The following arguments are supported:
+
+* `location` - (Required) The Azure Region where the Service Tags exists. This value is not used to filter the results, use `region` for that purpose instead. More information can be found here: [Service Tags URL parameters](https://docs.microsoft.com/en-us/rest/api/virtualnetwork/servicetags/list#uri-parameters).
+
+* `service` - (Required) The type of the service for which address prefixes will be fetched. Available service tags can be found here: [Available service tags](https://docs.microsoft.com/en-us/azure/virtual-network/service-tags-overview#available-service-tags).
+
+---
+
+* `location_filter` - (Optional) Changes the scope of the service tags. Can be any value that is also valid for `location`. 
```

Comment:
could we expand on this and talk about global vs regional service tags?

Reply:
adjusted documentation